web_redeploy.sh

```diff
@@ -1,7 +1,26 @@
 #! /bin/bash
+set -x
+
+ORIG_TEACHER_DASHBOARD_DIR=`grep -i "Alias /teacher-dashboard" ./docker-files/zeeguu-web/apache-zeeguu.conf | awk '{ print $3 }'`
+NEW_TEACHER_DASHBOARD_DIR="/opt/Zeeguu-Teacher-Dashboard-React/zeeguu-teacher-dashboard/build"
 
 docker restart zeeguu-web-dev
 
-#To consider this which is faster:
-#docker exec zeeguu-api-core-dev service apache2 reload
+#Start the python projects in develop mode
+docker exec zeeguu-web-dev python /opt/Zeeguu-Web/setup.py develop
+docker exec zeeguu-web-dev python /opt/Zeeguu-Exercises/src/setup.py develop
+docker exec zeeguu-web-dev python /opt/Zeeguu-Reader/src/setup.py develop
+docker exec zeeguu-web-dev python /opt/Zeeguu-Teacher-Dashboard/src/setup.py develop
+
+# Start the NodeJS projects in develop mode
+docker exec -d zeeguu-web-dev bash -c "cd /opt/Zeeguu-Teacher-Dashboard-React/zeeguu-teacher-dashboard && npm run start"
+docker exec -d zeeguu-web-dev bash -c "cd /opt/Zeeguu-Exercises && npm run dev"
```

Comment:
why is the first npm start and the next dev?

Reply:
this could be changed, the reasoning is that teacher dashboard was generated using create-react-app and that one autogenerates "start" to run the dev version of the project. but this can be changed if necessary